fvm/src/gas/price_list.rs

```diff
@@ -32,9 +32,117 @@ lazy_static! {
-        ipld_put_per_byte: 1,
+        create_actor_compute: 1108454,
+        create_actor_storage: 36 + 40,
+        delete_actor: -(36 + 40),
+
+        bls_sig_cost: 16598605,
+        secp256k1_sig_cost: 1637292,
+
+        hashing_base: 31355,
+        compute_unsealed_sector_cid_base: 98647,
+        verify_seal_base: 2000, // TODO revisit potential removal of this
+
+        verify_aggregate_seal_base: 0,
+        verify_aggregate_seal_per: [
+            (
+                RegisteredSealProof::StackedDRG32GiBV1P1,
+                449900
+            ),
+            (
+                RegisteredSealProof::StackedDRG64GiBV1P1,
+                359272
+            )
+        ].iter().copied().collect(),
+        verify_aggregate_seal_steps: [
+            (
+                RegisteredSealProof::StackedDRG32GiBV1P1,
+                StepCost (
+                    vec![
+                        Step{start: 4, cost: 103994170},
+                        Step{start: 7, cost: 112356810},
+                        Step{start: 13, cost: 122912610},
+                        Step{start: 26, cost: 137559930},
+                        Step{start: 52, cost: 162039100},
+                        Step{start: 103, cost: 210960780},
+                        Step{start: 205, cost: 318351180},
+                        Step{start: 410, cost: 528274980},
+                    ]
+                )
+            ),
+            (
+                RegisteredSealProof::StackedDRG64GiBV1P1,
+                StepCost (
+                    vec![
+                        Step{start: 4, cost: 102581240},
+                        Step{start: 7, cost: 110803030},
+                        Step{start: 13, cost: 120803700},
+                        Step{start: 26, cost: 134642130},
+                        Step{start: 52, cost: 157357890},
+                        Step{start: 103, cost: 203017690},
+                        Step{start: 205, cost: 304253590},
+                        Step{start: 410, cost: 509880640},
+                    ]
+                )
+            )
+        ].iter()
+        .cloned()
+        .collect(),
+
+        verify_consensus_fault: 495422,
+        verify_replica_update: 36316136,
+        verify_post_lookup: [
+            (
+                RegisteredPoStProof::StackedDRGWindow512MiBV1,
+                ScalingCost {
+                    flat: 117680921,
+                    scale: 43780,
+                },
+            ),
+            (
+                RegisteredPoStProof::StackedDRGWindow32GiBV1,
+                ScalingCost {
+                    flat: 117680921,
+                    scale: 43780,
+                },
+            ),
+            (
+                RegisteredPoStProof::StackedDRGWindow64GiBV1,
+                ScalingCost {
+                    flat: 117680921,
+                    scale: 43780,
+                },
+            ),
+        ]
+        .iter()
+        .copied()
+        .collect(),
```

Comment:
what’s the type you are collecting into here?

Reply:
I copy-pastad here, will check in an hour-ish.

Reply:
A map.